bundler.go

```diff
@@ -449,6 +452,16 @@ func (b *Bundler) bundle(e ConfigurationEnvironment) (err error) {
 		"TAGS=" + os.Getenv("TAGS"),
 	}
 
+	if e.CC != "" {
```

Comment:
Loop in the new `EnvironmentVariables` attribute and append environment variables accordingly instead.